src/home/index.html

```diff
@@ -52,5 +53,170 @@ <h1 class="heading">I am Mwafrika Glad to see you!</h1>
+
+      <section class="tonic">
+        <img src="../assert/images/tonic.png" alt="" />
+        <h1>Tonic</h1>
+        <div class="tonic-list">
+          <div class="black">CANOPY</div>
+          <div class="dot"></div>
+          <div class="gray">Back End Dev</div>
+          <div class="dot"></div>
+          <div class="gray">2015</div>
+        </div>
+        <p class="tonicDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <p>html</p>
+          <p>css</p>
+          <p>javascript</p>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>
+        </div>
+      </section>
+      <section class="tonic">
+        <img src="../assert/images/multiple2.png" alt="" />
+        <h1>Multi-Post Stories</h1>
+        <div class="tonic-list">
+          <div class="black">CANOPY</div>
+          <div class="dot"></div>
+          <div class="gray">Back End Dev</div>
+          <div class="dot"></div>
+          <div class="gray">2015</div>
+        </div>
+        <p class="tonicDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <p>html</p>
+          <p>css</p>
+          <p>javascript</p>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>
+        </div>
+      </section>
+      <section>
+        <h1 class="heading">About Myself</h1>
+        <p class="description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="subdescription">LET'S CONNECT</p>
+        <div class="social-container">
+          <ul>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/twitter.png" alt="Twitter" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/Linkedin.png" alt="LinkedIn" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/m.png" alt="Medium" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/github.png" alt="Github" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/fingers.png" alt="Finger" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>

```

Comment:
Please implement the button interactions (enable, hover, etc.). Also, i think as per[ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) we should write **Get my resume**  instead of "see project" .